### ISSUE TITLE
sshfs create file

### ISSUE THREAD

**Issue by altg1970** [PowerShell/Win32-OpenSSH#541]
Hi there,

I have installed the OpenSSH-Win64 in a windows server 2008 in order to make a clonezilla copy of fewe servers using ssh.
We have the openssh installed and working fine, we have tested it with putty, we can create dir and files, etc..
but if you want to use clonezilla, the tranfer fails because the clonezilla try to mount a sshfs file system, and it does ok and even can create directory tree but windows don't allow create files. 
we have checked the same behaviour using a opnsuse and mounting the unit with sshfs, you can create dir but not files.

have you tried what I said?

thanks for you help

**Comment by manojampalam** [comment]
Haven't worked with CloneZilla yet. It will help if you can explain how it works and how its related to SSH and SFTP

**Comment by altg1970** [comment]
Hello Manoj,

Really the problem isn´t related clanezilla itself, I detected this issue
because we use clonezilla for disk images  and Clonezilla can storage that
images in external usb drive, NFS, SAMBA and SSH. We need to use a windows
server as ssh server so we install your openssh. The connection and
autentication process between clonezilla and your openssh was fine the
problem showed up when clonezilla try to mount a unit using sshfs type well
really clonezilla can mount a unit using sshfs, and even create directories
but when the program try to create files the system deny the operation.
Clonezilla is a live CD so it tried to mount a file system in the target
that you select.

We wanted to be sure if it was clonezilla problem or it was sshfs mount
type so we used an opensuse and try to tested it and we had the same
problem, we were able to mount the unit, created directories but we weren't
able to create files. I don't know where to look up the sources of this
issue, may it be server policies, it seems windows fail but I don't know,
if you tell me where can I look up, log, events etc.. I can narrow the
source of the problem.

Regards

2017-03-01 22:32 GMT+01:00 Manoj Ampalam <notifications@github.com>:
